.github/workflows/primer_comment.yaml

```diff
@@ -24,6 +24,10 @@ jobs:
     name: Run
     runs-on: ubuntu-latest
     steps:
+      - name: Skip if PR run failed
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
is this the right way to check for failure?

Reply:
For some reason you can't see you only want to trigger this workflow if the dependent workflow succeeds. Only completed is supported.

Thus, even if Primer / PR fails this workflow gets triggered. This is a sort of quick stop so that you won't try and investigate why Primer / Comment failed but rather why Primer / PR fails.

Wait..... If we start failing Primer / Run for fatal warnings then we won't comment with this line anymore. Perhaps we should remove the top lines.

The lines added at the bottom are still valid though.

Reply:
I just would have thought you have to check for != success. Otherwise on every success, this exits with a failure code?

And yeah, I'll remove the huge projects like home-assistant and/or pandas later tonight if need be. No worries. Thanks for everything!